examples/teleport-usage/teleport_usage.py

```diff
@@ -0,0 +1,115 @@
+import json
+import datetime
+import boto3
+import time
+import os
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+# Check for AWS DynamoDB table name and region ENV, if none prompt
+TABLE_NAME = os.getenv("TABLE_NAME")
+if TABLE_NAME == None:
+    TABLE_NAME = input("DynamoDB events table name: ")
+
+aws_region = os.getenv("AWS_REGION")
+if aws_region == None:
+    aws_region = input("AWS Region for DynamoDB: ")
+
+# Check ENV for start and end times, if none, set last 30 days
+start_date = os.getenv("START_DATE")
+if start_date == None:
+    now = datetime.datetime.now()
+    start_date = now - datetime.timedelta(days=30)
+    
+end_date = os.getenv("END_DATE")
+if end_date == None:
+    end_date = now
+
+# Configure DynamoDB for query
+dynamodb = boto3.resource("dynamodb", region_name=aws_region)
+table = dynamodb.Table(TABLE_NAME)
+
+# variables for protocols
+app = 'app.session.chunk'
+db = 'db.session.start'
+kube = 'kube.request'
+ssh = 'session.start'
+win = 'windows.desktop.session.start'
+
+protocols = ['app', 'db', 'kube', 'win']

```

Comment:
I'm a little confused about the intent here. These are quoted - did you mean to use the variables above here? Or are you intentionally separating out `ssh` from this list and doing two loops because you want to specifically look for `session.start` and all events with type starting with one of `"app", "db", "kube", "win"`?

Reply:
And if you separated ssh into its own loop for a reason, could you clarify why?

Reply:
I separated SSH out because for some reason when I had them together the SSH portion would also return DB and Win users. Not sure if this is because they all have `session.start`. I did attempt to use `eq` rather than `begins_with` but yielded similar results.

Reply:
Hmm I'm looking at my own cluster's events in dynamodb and I don't see that when I query for `session.start` event type, all of my `FieldsMap.user` are my Teleport user login... Are you able to reproduce this? I'm interested in what happened there

Reply:
Yes I can, this time is does not return any SSH users. 

First attempt is normal. Second attempt is with just adding `ssh` to the list of protocols ( `protocols = ['ssh', 'app', 'db', 'kube', 'win']` ). Last attempt is with the added protocol and deleted code for first loop.

```
[ec2-user@ip-172-31-25-188 hello]$ /usr/local/bin/python3.9 /home/ec2-user/hello/x-all-teleport_usage.py
ssh: 7
app: 0
db: 1
kube: 0
win: 0

[ec2-user@ip-172-31-25-188 hello]$ /usr/local/bin/python3.9 /home/ec2-user/hello/x-all-teleport_usage.py
ssh: 7
ssh: 0
app: 0
db: 1
kube: 0
win: 0
 
[ec2-user@ip-172-31-25-188 hello]$ /usr/local/bin/python3.9 /home/ec2-user/hello/x-all-teleport_usage.py
ssh: 0
app: 0
db: 1
kube: 0
win: 0
```

Reply:
That would make sense because `EventType` won't start with `ssh`, it'll start with `session`